.github/CONTRIBUTING.md

```diff
@@ -34,67 +41,69 @@ A few minor points of style that you should adhere to in your PR:
+When you file a pull request, you should add some tests to this file with this in mind -- for new features, try to cover possible use cases extensively (we use [Codecov](https://app.codecov.io/gh/Rdatatable/data.table) to make it a bit easier to see how well you've done to minimally cover any new code you've added); for bug fixes, include a minimal version of the problem you've identified and write a test to ensure that your fix indeed works, and thereby guarantee that your fix continues to work as the codebase is further modified in the future. We encourage you to scroll around in `tests.Rraw` a bit to get a feel for the types of examples that are being created, and how bugs are tested/features evaluated.
 
-What numbers should be used for new tests? Numbers should be new relative to current master at the time of your PR. If another PR is merged before yours, then there may be a conflict, but that is no problem, as [Matt will fix the test numbers when he merges your PR](https://github.com/Rdatatable/data.table/pull/4731#issuecomment-768858134).
+What numbers should be used for new tests? Numbers should be new relative to current master at the time of your PR. If another PR is merged before yours, then there may be a conflict, but that is no problem, as [a Committer will fix the test numbers when merging your PR](https://github.com/Rdatatable/data.table/pull/4731#issuecomment-768858134).
 
 #### Using `test`
 
-See [`test` function manual](https://rdatatable.gitlab.io/data.table/reference/test.html).
+See [`?test`](https://rdatatable.gitlab.io/data.table/reference/test.html).
 
 **References:** If you are not sure how to issue a PR, but would like to contribute, these links should help get you started:
 
 1. **[How to Github: Fork, Branch, Track, Squash and Pull request](https://gun.io/blog/how-to-github-fork-branch-and-pull-request/)**.
-2. **[Squashing Github pull requests into a single commit](http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit)**.
-3. **[Github help](https://help.github.com/articles/using-pull-requests/)** - you'll need the *fork and pull* model.
+1. **[Squashing Github pull requests into a single commit](http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit)**.
+1. **[Github help](https://help.github.com/articles/using-pull-requests/)** - you'll need the *fork and pull* model.
```

Comment:
We have triple 1. in line prefixes

Reply:
yes that's intentional. note the rendered version:

![Screenshot_2023-12-30-15-47-46-32_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/Rdatatable/data.table/assets/7606389/a8cdc576-9b81-4cfd-b5e9-cab55fa205f2)

using repeated 1. lets the list be rearranged later without a huge distracting diff